pkg/bootstrap/bootstrap_config.go

```diff
@@ -325,7 +334,7 @@ func WriteBootstrap(config *meshconfig.ProxyConfig, node string, epoch int, pilo
 	setStatsOptions(opts, meta, nodeIPs)
 
 	// Support multiple network interfaces
-	meta["ISTIO_META_INSTANCE_IPS"] = strings.Join(nodeIPs, ",")
+	meta[model.NodeMetadataInstanceIPs] = strings.Join(nodeIPs, ",")
```

Comment:
Did you mean to make these changes here? Looks the same as some recent changes/PR

Reply:
I had discovered this bug just before #14735 was created. Once that lands, I'll rebase and this change will go away.